deep-learning/_posts/2017-04-10-deepmask.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: review
+title:  "Learning to Segment Object Candidates"
+tags:   deep-learning CNN segmentation
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1612.03144.pdf"
+cite:
+  authors: "Pinheiro, P. O., Collobert, R., & Dollar, P."
+  title:   "Learning to segment object candidates"
+  venue:   "Advances in Neural Information Processing Systems (pp. 1990-1998) (2015)"
+---
+
```

Comment:
Suggestion: give a brief overview of the purpose of the method before anything else. For example:

>This paper presents the DeepMask architecture, which can be used as an object detection method when combined with R-CNN.